### DISCUSSION TITLE
Version 1.30.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#794] [Announcements]
- Places search: allows you to search OpenStreetMap for places in the local area (thanks to @Sir-Photch)
- Search filters:
  - temporary filters to quickly limit or extend the categories that should included
  - includes an "online results" filter: as long as this filter is turned turned off, only local results are shown and no network requests are made, to protect your privacy
  - you can customize the default filter in settings
- More clock widget customizations:
  - added 1 more clock style (7-segment, thanks to @strongville)
  - added Material You option to every clock style (thanks to @strongville)
  - added an option to show seconds to every clock style (thanks to @strongville)
  - added the ability to use custom widgets as clock
- Widget customizations
  - app widgets now respect the launcher's color scheme (if the widget supports it); this can be toggled per widget.
  - app widgets can now be resized horizontally
- **⚠️ Breaking:** Removed HERE weather provider due to API / pricing changes. Please make sure to switch to a different provider after the update to continue receiving weather updates.

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.30.0'>Version 1.30.0</a>.</em>

**Comment by abonhote** [comment]
How is the map thing to be used? Maybe I'm too dumb, but I don't get it.

(Awesome launcher by the way, thanks a ton!)

**Reply by shtrophic** [reply]
You need to enable Places in the search Settings and Tap on "Online Results" in the Filter bar, after you entered your search query.

If that still does not work, maybe try setting the Overpass-Url to `http://overpass.osm.ch/api/`

**Reply by abonhote** [reply]
I missed the online results part. Thanks!

**Reply by TheSmilingTurtle** [reply]
This is probably my error, but i could not get the search to work I tried changing the api link because I am in switzerland, but to no avail. Any help would be appreciated. Also, is there a sure-fire way to get a result? (if it works, maybe I have just gotten unlucky with the search terms)

**Reply by shtrophic** [reply]
Hm, hard to say without knowing exactly whats going on, but here some things to consider:

- Don't increase the search radius past 1,5km, especially if you are from an urban region since the area to search will increase drastically which will also increase the time to get results.
- Make sure that you receive some GPS location. You can open a navigation app and if you see yourself on the map there, kvaesitso will also receive a location. (As long as the launcher has location permissions)
- Search for something where you know that it is inside your search radius. Where I am from, there are a lot of Pizza places so just typing "Pizza" works for me
- Make sure that you tap on online-results and that you have location results enabled
- Also try to switch back to the original API endpoint if nothing else works since OSM in Switzerland is way better than anywhere else which could also increase the search time, similar to search radius. (Also since the other fellow from this thread seemed to get it working w/o the switzerland API)
- You could also search for your and another location to search for in the launcher on osm.org while having your search radius reduced to the mininum, and disable "Hide uncategorized Results"

**Reply by TheSmilingTurtle** [reply]
Ok yea, turns out i had just gotten very unlucky with the searches, the tip to look it up along side osm.org helped. thank you :)

**Comment by neronguyen** [comment]
Cannot install from newest release apk #807

**Reply by exstntlmsnthrp** [reply]
That depends on a number of factors. Since you haven't provided any information, it's difficult for anyone to provide help. 

If you first installed it from Google Play and then try to install it from GitHub or Fdroid, then there will be a signature mismatch on the APK. You need to install it from the source where you originally installed it from or backup your correct settings (if that's important to you), uninstall the app, and then reinstall the new version.
